src/System.IO.MemoryMappedFiles/tests/MemoryMappedFile.CreateNew.Tests.cs

```diff
@@ -195,14 +200,23 @@ public void ReusingNames_Windows(string name)
             {
                 ValidateMemoryMappedFile(mmf, capacity);
             }
-            using (MemoryMappedFile mmf = MemoryMappedFile.CreateNew(mapName, capacity, access))
+
+            // MemoryMappedFileAccess.ReadExecute or MemoryMappedFileAccess.ReadWriteExecute isn't permitted inside an AppContainer
```

Comment:
NIT: should we just remove this comment and instead have this in the commit message? that way if people want to figure out why we added this they can go in the history.